Gruntfile.js

```diff
@@ -2,4 +2,29 @@ module.exports = function (grunt) {
 	require('grunt-dojo2').initConfig(grunt, {
 		/* any custom configuration goes here */
 	});
+
+	function setCombined(combined) {
+		if (combined) {
+			grunt.config('intern.options.reporters', [
+				{ id: 'tests/support/Reporter', file: 'coverage-unmapped.json' }
+			]);
+		}
+	}
+	setCombined(grunt.option('combined'));
+
+	grunt.registerTask('test', function () {
```

Comment:
Okay perhaps we can add an option to skip the coverage to the `grunt-dojo2` `test` task in the future.